gradle.properties

```diff
@@ -1 +1,15 @@
 org.gradle.configureondemand=true
+
+byteBuddyVersion=1.8.11
+credentialsPluginVersion=2.1.10
```

Comment:
Nit: Do we want to visually separate plugin and "regular" dependency versions into different blocks like we do e.g. in ORT?

Reply:
Good idea, done.